openwisp_radius/migrations/0001_initial_freeradius.py

```diff
@@ -1,26 +1,12 @@
-# Generated by Django 2.0.8 on 2018-09-27 11:44
-
-from django.conf import settings
-import django.core.validators
 from django.db import migrations, models
-import django.db.models.deletion
 import django.utils.timezone
-import django_freeradius.base.models
 import model_utils.fields
-import openwisp_radius.models
-import openwisp_users.mixins
-import re
-import uuid
 
 
 class Migration(migrations.Migration):
 
     initial = True
-
```

Comment:
I'd add some comments here (or why not a docstring to the class) explaining this is the default freeradius schema